Examples/js/earthquakeUSGS.js

```diff
@@ -16,13 +16,8 @@
             alias: "title",
             dataType: tableau.dataTypeEnum.string
         }, {
-            id: "lat",
-            alias: "latitude",
-            dataType: tableau.dataTypeEnum.float
-        }, {
-            id: "lon",
-            alias: "longitude",
-            dataType: tableau.dataTypeEnum.float
+            id: "location",
```

Comment:
Will do. Thanks. Just testing things right now to verify the doc changes. I'll make the doc changes next.